### DISCUSSION TITLE
can't run example

### DISCUSSION THREAD

**Discussion by Geliusan** [MarshalX/tgcalls#191] [Q&A] [answered]
can't run example device_playout.py 
I get this output:
Traceback (most recent call last):
  File "/home/geliusan/tgcalls2/tgcalls/examples/device_playout.py", line 15, in <module>
    CLIENT_TYPE = pytgcalls.GroupCallFactory.MTPROTO_CLIENT_TYPE.PYROGRAM
AttributeError: module 'pytgcalls' has no attribute 'GroupCallFactory'
I can't figure out how to fix it

**Comment by Geliusan** [comment, marked as answer]
Turns out it didn't build correctly, the problem was with the command: "pip3 install -U pytgcalls[pyrogram]" , it installs pytgcalls version 2.1.0 while tgcalls installs version 3.0.0.dev5
As a result, I used the command: "pip3 install pytgcalls[pyrogram]==3.0.0.dev5", an error occurred during installation. So I tried to install the latest version of pytgcalls: "pip3 install pytgcalls[pyrogram]==3.0.0.dev22". As a result, everything was installed without errors.

**Comment by MarshalX** [comment]
Added support for Pyrogram 2.0

pip install pytgcalls==3.0.0.dev22

Examples for dev versions: https://gist.github.com/MarshalX/3c99fcae532178928d93a7aae6f0efde

Check this if you cant install: https://github.com/MarshalX/tgcalls/discussions/44
